lib/rules/string-no-newline/__tests__/index.mjs

```diff
@@ -46,35 +50,35 @@ testRule({
 			message: messages.rejected,
 			line: 1,
 			column: 22,
-			endLine: 2,
-			endColumn: 5,
+			endLine: 1,
+			endColumn: 26,
```

Comment:
`bad-string-token` stops at the end of the line.
Before it would scan ahead to a subsequent quote mark.